onnxruntime/core/optimizer/attention_fusion_helper.h

```diff
@@ -675,7 +675,7 @@ bool MatchInputMaskSubgraph(const Graph& graph, const Node& qkv_matmul, Attentio
     if (!optimizer_utils::IsAttributeWithExpectedValue(qk_root, "alpha", 0.125f) ||
```

Comment:
It is not always 0.125. The value is 1.0 / sqrt( hidden_size / num_attention_heads). See the original code of checking Div constant.